salt/tests/requirements.in

```diff
@@ -1,4 +1,5 @@
-pytest ~= 4.3.0
+pytest
+pytest-cov
```

Comment:
Why not pin them? At least with `~=` or `>=` maybe?

Reply:
Honestly just don't really know which version we need but anyway it's not really needed to pin the version here since it's pinned in the `requirements.txt` and the requirements.txt is only updated manually (with pip-compile) so to me when it will break something with a newer `requirements.txt` then we should pin the version here